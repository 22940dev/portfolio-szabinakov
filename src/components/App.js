import React from 'react';
import {Switch, Route} from 'react-router-dom'

import Contact from './Contact'
import Navbar from './Navbar';
import Home from './Home'
import Footer from './Footer'

import '../styles/App.css'

function App() {

  return (
    <div className="App">
      <Navbar />
      <Switch>
        <Route exact path='/' component={Home}/>
        <Route path='/contact' component={Contact}/>
      </Switch>
      <Footer/>
    </div>
  );
}

export default App;
