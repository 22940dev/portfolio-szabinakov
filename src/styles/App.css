.App {
    font-family: 'Montserrat', sans-serif;
    background-color: #ECF8F8;
}