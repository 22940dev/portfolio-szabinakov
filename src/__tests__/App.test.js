import React from 'react';
import { render } from '@testing-library/react';
import App from '../components/App';
import { BrowserRouter } from 'react-router-dom';


describe('App', () => {
  it('matches the Snapshot', () => {

    const {asFragment} = render (
      <BrowserRouter>
        <App/>
      </BrowserRouter>)

    expect(asFragment()).toMatchSnapshot()
  })
})






// test('renders learn react link', () => {

//   const { getByText } = render(<App />);
//   const linkElement = getByText(/learn react/i);
//   expect(linkElement).toBeInTheDocument();
// });
