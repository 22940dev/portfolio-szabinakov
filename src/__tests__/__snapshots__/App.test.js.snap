// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches the Snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="menuWrap"
      data-testid="menuWrap"
    >
      <input
        class="toggler"
        data-testid="toggler"
        type="checkbox"
      />
      <div
        class="hamburger"
        data-testid="hamburger"
      >
        <div />
      </div>
      <div
        class="menu"
        data-testid="menu"
      >
        <div>
          <div>
            <ul>
              <li>
                <a
                  class="navbarListElement"
                  data-testid="navbarListElementHome"
                  href="/"
                >
                  Home
                </a>
              </li>
              <li>
                <a
                  class="navbarListElement"
                  data-testid="navbarListElementContact"
                  href="/contact"
                >
                  Contact
                </a>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div
      class="introContainer"
      data-testid="introContainer"
    >
      <div
        class="helloDiv"
        data-testid="helloDiv"
      >
        <h1>
          Hello,
        </h1>
      </div>
      <div
        class="myNameDiv"
        data-testid="myNameDiv"
      >
        <p
          class="myName"
        >
          My name is Szabina!
        </p>
      </div>
      <div
        class="aboutContainer"
        data-testid="aboutContainer"
      >
        and I am a Software Engineer.
      </div>
    </div>
    <div
      class="projectContainer"
      data-testid="projectContainer"
    >
      <div
        class="card"
        data-testid="card"
      >
        <div
          class="cardInner"
        >
          <div
            class="cardFront"
            data-testid="bringAndShareFront"
          >
            <h3>
              Bring and Share
            </h3>
            <div
              class="imgContainer"
            >
              <img
                alt="Bring and Share"
                src="BandS.png"
              />
            </div>
          </div>
          <div
            class="cardBack"
            data-testid="bringAndShareBack"
          >
            <p
              class="projectPhar"
            >
              This app has been a group project where I have been fully responsible to set up the API, database, and write tests. The App is allow you to create and event and add on participants with their diatry requriments. I have created the component for adding participants, and the participant list with all their details.
            </p>
            <div
              class="linksCard"
            >
              <a
                class="aCardLink"
                href="https://github.com/szabinakov/Bring-and-Share"
                target="blank"
              >
                Front-End
              </a>
              <a
                class="aCardLink"
                href="https://github.com/szabinakov/bring-and-share-api"
                target="blank"
              >
                Back-End
              </a>
              <a
                class="aCardLink"
                href="https://bring-and-share-final-project.vercel.app/"
                target="blank"
              >
                Website
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="card"
      >
        <div
          class="cardInner"
        >
          <div
            class="cardFront"
            data-testid="surrealFront"
          >
            <h3>
              Surreal Estate
            </h3>
            <div
              class="imgContainer"
            >
              <img
                alt=""
                src=""
              />
            </div>
          </div>
          <div
            class="cardBack"
            data-testid="surrealBack"
          >
            <p
              class="projectPhar"
            >
              It is a property search engine just like Zoopla or Rightmove. There are properties, favourites and add property pages.The user is able to log in with their Facebook account. Once logged in, you are able to save properties to your favourites or remove them.
            </p>
            <div
              class="linksCard"
            >
              <a
                class="aCardLink"
                href="https://github.com/szabinakov/surreal-estate"
                target="blank"
              >
                View Source
              </a>
              <a
                class="aCardLink"
                href="https://surreal-estate-eight.vercel.app/"
                target="blank"
              >
                Website
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="card"
      >
        <div
          class="cardInner"
        >
          <div
            class="cardFront"
            data-testid="weatherFront"
          >
            <h3>
              Weather App
            </h3>
            <div
              class="imgContainer"
            >
              <img
                alt="Weather App Project"
                src="weather.png"
              />
            </div>
          </div>
          <div
            class="cardBack"
            data-testid="weatherBack"
          >
            <p
              class="projectPhar"
            >
              By using external data the app allows the user to search by cities. Once you type in the requested city it displays 5 days of weather forecast. By clicking on More Details, it gives you a more detailed description of the weather of the chosen day.
            </p>
            <div
              class="linksCard"
            >
              <a
                class="aCardLink"
                href="https://github.com/szabinakov/weather-app"
                target="blank"
              >
                View Source
              </a>
              <a
                class="aCardLink"
                href="https://weather-app-ochre-two.vercel.app/"
                target="blank"
              >
                Website
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="card"
      >
        <div
          class="cardInner"
        >
          <div
            class="cardFront"
            data-testid="nasaFront"
          >
            <h3>
              Nasa's Picture Search
            </h3>
            <div
              class="imgContainer"
            >
              <img
                alt="Nasa Search Project"
                src="nasa.png"
              />
            </div>
          </div>
          <div
            class="cardBack"
            data-testid="nasaBack"
          >
            <p
              class="projectPhar"
            >
              This is my first React web application I have built. The idea is to search images based on a query realiting to space using the NASA API.
            </p>
            <div
              class="linksCard"
            >
              <a
                class="aCardLink"
                href="https://github.com/szabinakov/Nasa-picture-searchengine"
                target="blank"
              >
                View Source
              </a>
              <a
                class="aCardLink"
                href="https://tech-test.vercel.app/"
                target="blank"
              >
                Website
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="card"
      >
        <div
          class="cardInner"
        >
          <div
            class="cardFront"
            data-testid="cruiseFront"
          >
            <h3>
              Cruise Ship
            </h3>
            <div
              class="imgContainer"
            >
              <img
                alt="Cruise Ship Project"
                src="cruise.png"
              />
            </div>
          </div>
          <div
            class="cardBack"
            data-testid="cruiseBack"
          >
            <p
              class="projectPhar"
            >
              At this project I have built a cruise ship which is able to sail. There is a board also included which tells you the current and the next port. You are also able to add ports. It has been designed to look really oldschool. Because that is what I love the most.
            </p>
            <div
              class="linksCard"
            >
              <a
                class="aCardLink"
                href="https://github.com/szabinakov/cruise-ships"
                target="blank"
              >
                View Source
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="card"
      >
        <div
          class="cardInner"
        >
          <div
            class="cardFront"
            data-testid="petFront"
          >
            <h3>
              Virtual Pet
            </h3>
            <div
              class="imgContainer"
            >
              <img
                alt="Virtual Pet Project"
                src="pet.png"
              />
            </div>
          </div>
          <div
            class="cardBack"
            data-testid="petBack"
          >
            <p
              class="projectPhar"
            >
              I have built a Virtual Pet with JavaScript. Same principles like the good old game Tamagotchi. The Pet gets older and as it gets older, it gets hungrier and more and more unfit. You are able to take it for a walk to get back its fitness and of course feed it. However, nature is cruel and even if you take care of it on the best way the pet will D-I-E when it reaches 30 years.
            </p>
            <div
              class="linksCard"
            >
              <a
                class="aCardLink"
                href="https://github.com/szabinakov/Virtual-Pet"
                target="blank"
              >
                View Source
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
    <footer>
      <ul
        class="footerContainer"
      >
        <li
          class="footerListElement"
          data-testid="footerTestOne"
        >
          <a
            href="https://twitter.com/szabinkov?lang=en-gb"
            target="blank"
          >
            <img
              alt="twitter"
              href="www.google.com"
              src="twitter.png"
            />
          </a>
        </li>
        <li
          class="footerListElement"
          data-testid="footerTestTwo"
        >
          <a
            href="https://www.instagram.com/szabinush/"
            target="blank"
          >
            <img
              alt="instagram"
              src="insta.png"
            />
          </a>
        </li>
        <li
          class="footerListElement"
          data-testid="footerTestThree"
        >
          <a
            href="https://www.linkedin.com/in/szabina-kov%C3%A1cs-7a0592159/"
            target="blank"
          >
            <img
              alt="linkedin"
              src="linkedin.png"
            />
          </a>
        </li>
        <li
          class="footerListElement"
          data-testid="footerTestFour"
        >
          <a
            href="https://github.com/szabinakov"
            target="blank"
          >
            <img
              alt="Github"
              src="github.png"
            />
          </a>
        </li>
        <li
          class="footerListElement"
          data-testid="footerTestFive"
        >
          <a
            href="mailto:szabinkov@gmail.com"
          >
            <img
              alt="Gmail"
              src="gmail.png"
            />
          </a>
        </li>
      </ul>
    </footer>
  </div>
</DocumentFragment>
`;
